docker-compose.yml

```diff
@@ -15,36 +15,16 @@ services:
     image: nginx
     hostname: 'nginx'
     container_name: 'nginx'
+    depends_on:
+      - 'ferretdb'
```

Comment:
It does not wait for container to be ready, so in that case it does nothing

Reply:
I try adding a healthcheck command and send a ping to check if it's up.

Reply:
I used `pg_isready` instead, it seemed like the correct approach.

Reply:
I think we should just remove it – nginx does not depend on PostgreSQL at all, and FerretDB reconnects automatically

Reply:
I kept getting 502 errors from nginx without that healthcheck. But running `docker compose restart` after building the app resolves the issue.

So I will remove it.

Reply:
@AlekSi PTAL.